bot/utils/role_checks.py

```diff
@@ -0,0 +1,11 @@
+from discord.ext import commands
+
+
+async def _has_role_check(ctx: commands.Context, guild, *role_name: int) -> bool:
```

Comment:
```suggestion
async def _has_role_check(ctx: commands.Context, guild, *role_names: str) -> bool:
```

It looks like role names are strings, not ints. Also, since this is a varargs, it should be named plurally.